src/main/java/duke/command/CommandNewTask.java

```diff
@@ -22,6 +23,12 @@ public CommandNewTask(String userInput) {
 
     @Override
     public void execute(TaskList taskList) {
+        try {
```

Comment:
Should we really be adding this requirement for all tasks? Perhaps it would be better if the try-catch statement was inserted in the FDuration Class instead since it is a specific requirement.